### PR TITLE
add title and color to birthday calendar

### PR DESCRIPTION
**Pull request by georgehrke**
@DeepDiver1975 @jancborchardt

### CODE REVIEW

**Comment by georgehrke** [comment]
fixes https://github.com/owncloud/calendar/issues/289

File: apps/dav/appinfo/application.php [hunk 1]
```diff
@@ -124,7 +124,9 @@ public function __construct (array $urlParams=array()) {
 			/** @var IAppContainer $c */
 			return new BirthdayService(
 				$c->query('CalDavBackend'),
-				$c->query('CardDavBackend')
+				$c->query('CardDavBackend'),
+				//TODO - what app to use for location?
```
Comment (georgehrke) [inline apps/dav/appinfo/application.php]: it should obviously say translations and not location ^^
Todo should be resolved before merging anyway

File: apps/dav/lib/caldav/birthdayservice.php [hunk 2]
```diff
@@ -91,6 +94,11 @@ public function onCardDeleted($addressBookId, $cardUri) {
 	 * @throws \Sabre\DAV\Exception\BadRequest
 	 */
 	public function ensureCalendarExists($principal, $id, $properties) {
+		$properties = array_merge([
+			'{DAV:}displayname' => $this->l10n->t('Contact birthdays'),
```
Comment (DeepDiver1975) [inline apps/dav/lib/caldav/birthdayservice.php]: Translating the displayname is kind of difficult - this language is to be taken?

In most cases the user's language preference might not be set in the db - never the less the client device might still use a different language.

Comment (DeepDiver1975) [inline apps/dav/lib/caldav/birthdayservice.php]: > In most cases the user's language preference might not be set in the db

but at least we can take that one

Comment (DeepDiver1975) [inline apps/dav/lib/caldav/birthdayservice.php]: We can use a mechanism similar to https://github.com/owncloud/activity/blob/master/lib/backgroundjob/emailnotification.php#L117-L132

**Comment by DeepDiver1975** [comment]
I dropped the l10n support for now to have a minimal solution with allows backport for 9.0.1

**Comment by georgehrke** [comment]
@jancborchardt Are you happy with the default color? If not speak up please :)

**Comment by karlitschek** [comment]
please backport :+1:

**Comment by DeepDiver1975** [comment]
:+1:

**Pull request merged**